client/state/selectors/get-site-computed-attributes.js

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
In my opinion, a selector should only exist if it provides some practical value to be used independently in the application, or maybe at least used by more than one selector. A compromise here could be to have separate functions within a `get-site.js` selector file, with named exports for testing / composibility, but the default export as the main `getSite` selector.